contrib/base.sh

```diff
@@ -233,9 +245,21 @@ if [ -z "$SHA256_PROG" ]; then
     fail "Please install sha256sum or gsha256sum"
 fi
 
+export SORT_PROG=`which sort || which gsort`
```

Comment:
Hey this happens to work on macOS with sort -- but in the future it occurred to me we may want to prioritize `gThing` over `Thing`.  Chances are some systems have both and the `g*` version is the one we want. Just a note to self and also a note to you.. :)